### PR TITLE
feat: Setup shared workflow for acceptance test

### PR DESCRIPTION
**Pull request by ankitKumar-extendaRetail**

### CODE REVIEW

File: .github/workflows/slp-common-acceptance-test.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        uses: extenda/actions/setup-gcloud@v0
+        id: gcloud
+        with:
+          service-account-key: ${{ inputs.gcloud-auth-staging }}
+
+      - name: Identity token
+        uses: extenda/actions/identity-token@v0
+        with:
+          service-account-key: ${{ inputs.gcloud-auth-staging }}
+          service-account: basketauditfrontend@${{ steps.gcloud.outputs.project-id }}.iam.gserviceaccount.com
```
Comment (pallavibose) [inline .github/workflows/slp-common-acceptance-test.yml:51]: steps.gcloud.outputs.project-id why is this dynamic? Are we expecting diff env. 
If so then line 50 has inputs.gcloud-auth-**staging**. Wven that should be dynamic

Comment (pallavibose) [reply .github/workflows/slp-common-acceptance-test.yml:51]: Just rename the variable to gcloud-auth

**Review by pallavibose — CHANGES_REQUESTED** [review]

**Review by pallavibose — APPROVED** [review]
@ankitKumar-extendaRetail how can we add version support to this?

**Review by pallavibose — CHANGES_REQUESTED** [review]
Please check how to add versioning support

**Comment by ankitKumar-extendaRetail** [comment]
> @ankitKumar-extendaRetail how can we add version support to this?

Discussed

**Review by TwanVB — APPROVED** [review]

**Review by pallavibose — APPROVED** [review]

**Pull request merged**